pypose/module/icp.py

```diff
@@ -0,0 +1,74 @@
+import torch
+from . import EPnP
+from .. import LieTensor
+class ICP(torch.nn.Module):
+    r'''
+    Iterative Closest Point (ICP) using Singular Value Decomposition (SVD).
+    Args:
+        steplim: the max step number
+        tol: the tolerance of error for early stopping
+        matched: whether the input points set and the target points set have been matched
+    '''
```

Comment:
add doc rst definition, refer to [here](https://github.com/pypose/pypose/blob/main/CONTRIBUTING.md#12-writing-documentation)